packages/core/src/js/tracing/reactnavigation.ts

```diff
@@ -395,3 +372,12 @@ interface NavigationContainer {
   addListener: (type: string, listener: () => void) => void;
   getCurrentRoute: () => NavigationRoute;
 }
+
+/**
+ *
```

Comment:
n: We could add a comment in the generated stub

```suggestion
 * Returns React Navigation integration of the given client.
```